### PR TITLE
Docs/condo/sberdoma 612/readme refactoring

### PR DESCRIPTION
**Pull request by AntonAL**
Made documentation easier to read with added structure.
Added checklist before making a pull-request.

### CODE REVIEW

File: apps/condo/README.md [hunk 1]
```diff
@@ -1,142 +1,8 @@
...
-With that configs you can easily use `debugger` or IDE breakpoints to debug.
-Check `package.json` for debug command (`node inspect`).
+Condo development guide
+=====
+
+[Quick start](./docs/quick-start.md)
+[Project structure](./docs/project-structure.md)
+[Migrations guide](./docs/migrations.md)
+[Utils](./docs/utils)
+[Checklists](./docs/checklists)
```
Comment (toplenboren) [inline apps/condo/README.md:8]:
```suggestion
* [Quick start](./docs/quick-start.md)
* [Project structure](./docs/project-structure.md)
* [Migrations guide](./docs/migrations.md)
* [Utils](./docs/utils)
* [Checklists](./docs/checklists)
```

File: apps/condo/domains/common/components/icons/UserIcon.tsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import React from 'react'
```
Comment (toplenboren) [inline apps/condo/domains/common/components/icons/UserIcon.tsx:1]: I don't think this belongs here... 🚨 🚨

Comment (AntonAL) [reply apps/condo/domains/common/components/icons/UserIcon.tsx]: Yes, it was others commit.
Rebased now

**Review by dkoviazin — APPROVED** [review]

File: apps/condo/README.md (hunk 1 above)
Comment (AntonAL) [reply apps/condo/README.md]: Added in `de9ad365`

File: apps/condo/docs/checklists/pull-request.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+Sync TypeScript declarations for GraphQL entities:
+
+```shell
+yarn workspace @app/condo maketypes
+```
+
+Check imports, that should be from absolute paths, like following:
+
+```jsx
+import { something } from '@app/condo/domains/…' // correct
```
Comment (pahaz) [inline apps/condo/docs/checklists/pull-request.md:27]: It’s better to move it inside glabal style.md

**Review by pahaz — APPROVED** [review]

File: apps/condo/docs/checklists/pull-request.md (hunk 3 above)
Comment (pahaz) [reply apps/condo/docs/checklists/pull-request.md:27]: Or move it here

Comment (pahaz) [reply apps/condo/docs/checklists/pull-request.md:27]: It’s better to have one place for style docs

Comment (AntonAL) [reply apps/condo/docs/checklists/pull-request.md:27]: Ok, next iteration of working with docs will dedicate for coding style.

**Pull request merged**